for Sept 2025 publication — wcag3-intro.md (#1485)
w3c/wai-website: pages/standards-guidelines/wcag/wcag3-intro.md

{% include box.html type="start" h="2" title="Summary" class="full" %}
{:/}

Many aspects of W3C Accessibility Guidelines (WCAG) 3 are in an exploratory or developng phase and will change substantially.

> WCAG 3 will have similar fundamental and specific accessibility requirements as WCAG 2. WCAG 3 will have a different structure, different conformance model, and broader scope.

Reviewer: Rereading this, I would now change it to simply say "WCAG 3 will have similar accessibility requirements as WCAG 2."  We are not using "fundamental" any longer and I don't think "specific" adds anything.

> WCAG 3 will have similar fundamental and specific accessibility requirements as WCAG 2. WCAG 3 will have a different structure, different conformance model, and broader scope.

Author: ```suggestion
WCAG 3 will have similar accessibility requirements as WCAG 2. WCAG 3 will have a different structure, different conformance model, and broader scope.
```

{% include box.html type="start" h="2" title="Summary" class="full" %}
{:/}

> Many aspects of W3C Accessibility Guidelines (WCAG) 3 are in an exploratory or developng phase and will change substantially.

Author: ```suggestion
Many aspects of W3C Accessibility Guidelines (WCAG) 3 are in an exploratory or developing phase and will change substantially.
```

* Refining
* Mature

For more information on these status indicators, see the Explainer section [Current Process for Creating WCAG 3.0](https://www.w3.org/TR/wcag-3.0-explainer/#current-process).

### September 2025 update

> This update includes potential guidelines, requirements, and assertions that have progressed to Developing status.

Reviewer: Please add something like: "Guidelines, requirements and assertions that are still Exploratory are available in the Editor's Draft." with link.

Author: +1

I changed it to: "This update includes potential guidelines, requirements, and assertions that have progressed to Developing status. Those that are still Exploratory are available in the _Editor's Draft_." I think not repeating "guidelines, requirements, and assertions" is a bit easier to process. Feel free to re-open/comment if you suggest otherwise.

WCAG 3 will not supersede WCAG 2 and WCAG 2 will not be deprecated for at least several years after WCAG 3 is finalized.

> The Accessibility Guidelines Working Group (AG WG) has created an initial set of guidelines. In the first part of 2025, AG WG focused on progressing more guidelines, requirements, assertions, and supporting material to Developing status. In September 2025, the Editors will focus on drafting more guidelines, requirements, and assertions. We will use that experience to develop a projected timeline for WCAG 3. We expect the timeline to be available by December 2025.

Author: ```suggestion
The Accessibility Guidelines Working Group (AG WG) created an initial set of guidelines and explored conformance models. In 2025, AG WG has focused on progressing guidelines, requirements, assertions, and supporting material to Developing status. During the rest of 2025, we will focus on completing the proposed guidelines and a proposed conformance model for public review. AG WG will develop a projected timeline for WCAG 3. We expect the timeline to be available by December 2025.
```

> 

Reviewer: I think the tenses on this are incorrect. I also disagree with calling out the editor's meeting as just our work. The editor's will be pulling the greater group's work and discussions together and as such the group as a whole should be referenced.  Please consider:

In the first part of 2025, the Accessibility Guidelines Working Group (AG WG) created an initial set of guidelines. AG WG has since focused on progressing guidelines, requirements, assertions, and supporting material to Developing status. The rest of 2025 will focus on completing the set of developing guidelines and proposing a conformance model for public review.  AG WG will also develop a projected timeline for WCAG 3. We expect the timeline to be available by December 2025.

Author: I tweaked to "The Accessibility Guidelines Working Group (AG WG) previously created an initial set of guidelines and explored conformance models. In 2025, AG WG focused on progressing guidelines, requirements, assertions, and supporting material to Developing status. During the rest of 2025, the group will focus on completing the proposed guidelines and proposed conformance model for public review."

I'm hesitant to start with "In the first part of 2025, the Accessibility Guidelines Working Group (AG WG) created..." because it could be interpreted that all of that work happened only in 2025.

I simplified the timeline info in the next paragraph: "AG WG plans to develop a projected [WCAG 3 timeline](https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline) by December 2025."

Feel free to re-open/comment if you have additional edits.

WCAG 3 will not supersede WCAG 2 and WCAG 2 will not be deprecated for at least several years after WCAG 3 is finalized.

The Accessibility Guidelines Working Group (AG WG) previously created an initial set of guidelines and explored conformance models. In 2025, AG WG focused on progressing guidelines, requirements, assertions, and supporting material to Developing status. During the rest of 2025, the group will focus on completing the proposed guidelines and proposed conformance model for public review.

> AG WG plans to develop a projected [WCAG 3 timeline](https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline#Publication_Plan) by December 2025.

Author: ```suggestion
AG WG plans to develop a projected [WCAG 3 timeline](https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline) by December 2025.
```
